docs/manual/forms/form_fields.rst

```diff
@@ -211,6 +211,68 @@ Stel er zijn 4 velden:
   Er is nu een formulier gemaakt waarbij de straat en de stad automatisch worden
   ingevuld als de postcode en het huisnummer zijn ingevuld.
 
+Validatie
+---------
+
+* **Reguliere expressie**: een patroon waar de veldwaarde aan moet voldoen. Reguliere
+  expressies zijn een soort van programmeertaal die snel erg complex kunnen worden. Op
+  regex101_ kun je deze uitproberen.
+
+  We hebben een aantal kant-en-klare patronen die je als inspiratie kan gebruiken.
+
+  ================ ===================== ===============================================
+  Expressie        Voorbeeldwaarde       Toelichting
+  ================ ===================== ===============================================
+  ``[0-9]*``       0123                  Een onbeperkt aantal getallen
+  ``\d*``          0123                  Een onbeperkt aantal getallen - ``\d`` is
+                                         equivalent aan ``[0-9]``.
+  ``[0-9]{1,3}``   42                    Een tekst van minimaal 1 en maximaal 3
+                                         posities, waarbij elk karakter een getal moet
+                                         zijn
+  ``[a-zA-Z]``     X                     Eén (hoofdletterongevoelige) ASCII letter
+  ``[\w]{1,5}``    aF\_4                 1 tot 5 alfanumerieke karakters of liggend streepje
```

Comment:
since above you mentioned that `\d` is equivalent to `[0-9]`, maybe here you can say that `\w` is equivalent to `[a-zA-Z0-9_]`?